Ui/WinForms/Default.cs

```diff
@@ -40,11 +40,17 @@ public static bool ConfirmCallback(ISshKey key, Process process)
                 programName = string.Format("{0} ({1})", process.MainWindowTitle,
                     process.ProcessName);
             }
-            var result = MessageBox.Show(
+
+            /**
+             * MessageBox options for topmost and focus from:
+             * https://msdn.microsoft.com/en-us/library/windows/desktop/ms645505(v=vs.85).aspx
+             */
+
+            DialogResult result = MessageBox.Show(
                 string.Format(Strings.askConfirmKey, programName, key.Comment,
                 key.GetMD5Fingerprint().ToHexString()), Util.AssemblyTitle,
                 MessageBoxButtons.YesNo, MessageBoxIcon.Question,
-                MessageBoxDefaultButton.Button2
+                MessageBoxDefaultButton.Button2, (MessageBoxOptions)0x51000
```

Comment:
1000L is for MB_SYSTEMMODAL.
I must make some tests, I tested building in another machine and is not working, but the already built PLGX file I have in this machine is working, so I think this must have an older version of the fix (I was trying different combination of flags). So, let me try to decompile this PLGX file to see what happened, and I'll update the pull request.